projects/addon-editor/components/editor-new/editor-new.component.ts

```diff
@@ -135,4 +143,11 @@ export class TuiEditorNewComponent
     private get hasValue(): boolean {
         return !!this.value;
     }
+
+    private getMarkedLinkBeforeSelectClosest() {
```

Comment:
No return type, how did this go through precommit? :D

Reply:
fixed